docs/best-practices/pingone/index.md

```diff
@@ -0,0 +1,248 @@
+
+Example / bootstrapped configuration may be useful as a starting point when initially creating use cases with the service (in the development phase), but will create conflicts when migrating the configuration through to test and production environments.
+
+The definition of the example / bootstrapped configuration for new environment may also change over time, as new features are released and use case configuration best practices are defined.  Therefore an environment created today may not be the same as an environment created a year from now.
+
+It is best practice therefore to, when creating new environments other than the initial development environment, create them as "clean" environments, without example / bootstrapped configuration.  Currently, example / bootstrapped configuration must be manually removed from an environment.
+
+Notable examples include:
+
+#### Platform
+- The default branding theme
+- Optional directory schema attributes (which can be disabled if not used)
+  - `accountId`
+  - `address`
+  - `email`
+  - `externalId`
+  - `locale`
+  - `mobilePhone`
+  - `name`
+  - `nickname`
+  - `photo`
+  - `preferredLanguage`
+  - `primaryPhone`
+  - `timezone`
+  - `title`
+  - `type`
+- The default Keys and Certificates
+- The default notification policies
+- The default `Single_Factor` sign-on policy
+- The example password policies
+- The `PingOne Application Portal` (which can be disabled if not used)
+
+#### DaVinci service
+- Example Forms
+
+#### MFA service
+- The default MFA Device Policy
+- The default FIDO2 policies
+- The `Multi_Factor` sign-on policy
+
+#### Verify service
+- The default verify policy
+
+### Define All Configuration Dependencies in Terraform (or elsewhere in the Pipeline)
+
+Example / bootstrapped configuration is deployed automatically by the PingOne service when an environment is created (or new services are provisioned to an existing environment).  This is the default behaviour of the web admin console, and the API.
+
+Example / bootstrapped configuration may be useful as a starting point when initially creating use cases with the service (in the development phase), but will create conflicts when migrating the configuration through to test and production environments.
+
+The definition of the example / bootstrapped configuration for new environment may also change over time, as new features are released and use case configuration best practices are defined.  Therefore an environment created today may not be the same as an environment created a year from now.
+
+It is best practice therefore to, after developing flows for use cases, to explicitly define all configuration dependencies in Terraform (or as a prior step in the CICD pipeline).  Most notably, this includes defining the policies (e.g. sign-on, MFA Device, FIDO2, Protect policies) that applications will use in HCL, rather than using the example / bootstrapped environment examples.
```

Comment:
Therefore, It is best practice to explicitly define all configuration dependencies in Terraform (or as a prior step in the CICD pipeline) after developing flows for use cases.  Most notably, this practice includes ...